riscv/zicfiss.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _RISCV_ZICFISS_H
```

Comment:
Should add a comment like "See LICENSE for license details." so the license of this new file is clear.